include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -6901,10 +6901,11 @@ namespace VectorTools
                   const InVector        &fe_function,
                   const Point<spacedim>      &point)
   {
+#ifdef DEBUG
     const FiniteElement<dim> &fe = dof.get_fe();
-
     Assert(fe.n_components() == 1,
            ExcMessage ("Finite element is not scalar as is necessary for this function"));
+#endif
```

Comment:
For future cases like this, if `fe` is not used anywhere else, can we just inline the call to `dof.get_fe()` into the assertion itself? This way we don't have to obfuscate the code with `#ifdef...#endif`.